PQAnalysis/traj/trajectory.py

```diff
@@ -143,6 +143,71 @@ def __iter__(self) -> Iterable[AtomicSystem]:
             frame.topology = self.topology
             yield frame
 
+    def window(self, window_size: int, window_gap: int = 1, window_start: int = 0, window_stop: int = 0) -> Iterable[Iterable[AtomicSystem]]:
```

Comment:
a bit misleading, beacause also the __docs__ says that it is zero by default

```suggestion
    def window(self,
               window_size: int,
               window_gap: int = 1,
               window_start: int = 0,
               window_stop: int | None = None
               ) -> Iterable[Iterable[AtomicSystem]]:
```

Reply:
also `Iterable[Iterable[AtomicSystem]]` should give more information on the datatype - i would recommend to use `Iterable[List[AtomicSystem]]`, because it effecitvely returns a `List` and not any other `Iterable`. Then the use knows immediately which instance methods are available e.g. `.append()`.


```suggestion
    def window(self,  
            window_size: int,  
            window_gap: int = 1,  
            window_start: int = 0,  
            window_stop: int | None = None  
            ) -> Iterable[List[AtomicSystem]]:
```